driver/src/main/scala/api/cursor.scala

```diff
@@ -185,48 +232,106 @@ object DefaultCursor {
       @inline
       private def makeIterator(response: Response) = ReplyDocumentIterator(pack)(response.reply, response.documents)
 
-      @inline
-      private def makeRequest(implicit ctx: ExecutionContext): Future[Response] =
-        Failover2(mongoConnection, failoverStrategy) { () =>
-          mongoConnection.sendExpectingResponse(RequestMaker(query, documents, readPreference), isMongo26WriteOp)
-        }.future
+    @inline
+    private def makeRequest(implicit ctx: ExecutionContext): Future[Response] =
+      Failover2(mongoConnection, failoverStrategy) { () =>
+        mongoConnection.sendExpectingResponse(RequestMaker(query, documents, readPreference), isMongo26WriteOp)
+      }.future
+
+    @inline
+    private def isTailable =
+      (query.flags & QueryFlags.TailableCursor) == QueryFlags.TailableCursor
+
+    /** Returns next response using tailable mode */
+    private def tailResponse(current: Response, maxDocs: Int)(implicit context: ExecutionContext): Option[Future[Response]] =
```

Comment:
From https://github.com/ReactiveMongo/ReactiveMongo/blob/master/driver/src/main/scala/api/cursor.scala#L210 .
I no longer pass the `maxDocs` in the loop (`CustomEnumerator.SEnumerator((response, maxDocs))`) as I didn't see it used as `._2` there.